README.md

```diff
@@ -37,82 +38,77 @@ numerous other AWS resources.
 
-VPCs do not share RFC 1918 space, but keeping cidr blocks unique makes VPC
-peering easier should we want to do it in the future. The semantics of how we
-split clusters across VPCs and how much we care about unique cidr blocks may
-change in the future.
+It's easier if your IAM cluster manager account username doesn't match the one
+you'd like to use to SSH into your clusters. If your name is "Jane Smith", your
+IAM cluster manager user might be "janesmith" while your stack SSH username
+would be "jane".
 
-### Step 3 - Install mh-opsworks
+### Step 2 - Install mh-opsworks
 
 You must have ruby 2 installed, ideally through something like rbenv or rvm,
 though if your system ruby is >= 2 you should be fine. `./bin/setup` installs
-prerequisites and sets up empty cluster configuration templates.
+prerequisites and sets up an empty `secrets.json`.
 
-    git clone git@github.com:harvard-dce/mh-opsworks.git mh-opsworks/
+    git clone https://github.com/harvard-dce/mh-opsworks mh-opsworks/
     cd mh-opsworks
     ./bin/setup # checks for dependencies and sets up template env files
 
 The base scripts (`rake`, mostly) live in `$REPO_ROOT/bin` and all paths below
 assume you're in repo root.
 
-### Step 4 - Create your configuration files.
+### Step 3 - Create your configuration files.
 
-You can use multiple configuration files (more on that later), but by default
-`mh-opsworks` reads from `$REPO_ROOT/cluster_config.json` and
-`$REPO_ROOT/secrets.json`. If you're working with a cluster that already
-exists, get the correct `cluster_config.json` from the right person.
+First, be sure you have a `secrets.json` with the correct values in it. The
+most important ones are `access_key_id`, `secret_access_key`,
+`shared_asset_bucket_name`, and `cluster_config_bucket_name`. This file lives
+in the root of the repository by default.
 
-    cd mh-opsworks
+Once you've set up your `secrets.json` correctly, you can start working with
+clusters.
+
+If you'd like to work in an existing cluster, run:
+
+    ./bin/rake cluster:switch
 
-    # edit cluster_config.json with your specific values
-    vim cluster_config.json
+If you'd like to create a new cluster entirely, run:
 
-    # Edit secrets to include the correct AWS (and other) credentials.
-    vim secrets.json
+    ./bin/rake cluster:new
 
-### Step 5 - Sanity check your cluster configuration
+and follow the prompts.
+
+Be sure to set up the "users" stanza with your desired SSH username, rights,
+and public key.  Following the example set in Step 1, it'd be "jane".
+
+It's easiest if your SSH user matches your default local unix username as
+the `stack:instances:ssh_to` rake task will work out of the box.
+
+### Step 4 - Sanity check your cluster configuration
 
 We've implemented a set of sanity checks to ensure your cluster configuration
 looks right. They are by no means comprehensive, but serve as a basic
 pre-flight check. The checks are run automatically before every `rake` task.
```

Comment:
"except for those that deal with creating or administering cluster configs" <-- double check this